app/src/main/java/org/mozilla/webmaker/activity/Play.java

```diff
@@ -44,6 +62,20 @@ protected void onCreate(Bundle savedInstanceState) {
     }
 
     @Override
+    public boolean onPrepareOptionsMenu(Menu menu) {
+        MenuItem edit_item = menu.findItem(R.id.action_edit);
+
+        if (edit_item == null || login_username == null)
```

Comment:
Nitpick: Use `{}` consistently. The style below is what we generally use throughout the app.